README.md

```diff
@@ -182,27 +209,26 @@ The section for showing the animaton displays the shapes, as specified in the co
 * FadeOut
 * GrowFromCenter
 
-**Adding:**
-These are some of the methods for adding mobjects or playing Animations on mobjects. Note: If you play an animation, you don't have to add it to the screen. The animation does it for you. 
+**Adding:** These are some of the methods for adding mobjects or playing Animations on mobjects. Note: If you play an animation, you don't have to add it to the screen. The animation does it for you. 
 * Play
 * Add
 
 In this code, I specifically included an example that I found useful to know. 
-``` python
```

Comment:
This was changed in another commit. That has been taken into account. Ignore this.